tests/utils/exec/rosacli/pr_check.go

```diff
@@ -30,22 +32,37 @@ func GetCommitFoucs() (string, error) {
 		return "", err
 	}
 	fmt.Printf("\nThe last commit is: %s\n", output.String())
-	theStrSlice := strings.Split(output.String(), " ")
-
-	var tcIDs []string
-	reg := regexp.MustCompile(`(id:.*)`)
-	for _, theStr := range theStrSlice {
-		m := reg.FindAllString(theStr, -1)
-		if len(m) > 0 {
-			for _, idStr := range m {
-				idStr = strings.Split(idStr, "id:")[1]
-				ids := strings.Split(idStr, ",")
-				tcIDs = append(tcIDs, ids...)
-			}
-		}
+	return output.String(), err
+}
+
+func GetFocusCaseIDs(commitMessage string) (string, error) {
+	reg := regexp.MustCompile(`ids?:([0-9,\s,]*)`)
```

Comment:
It is not an excellent choice to relax the rule to match the space. We should force the user to follow the format `ids?:1234,1234`.

Reply:
I would respect team members habit.